src/config/config.ts

```diff
@@ -50,48 +41,25 @@ export class RaftConfig {
     return this._subgraphEndpoint;
   }
 
-  static getTokenAddress<T extends Token>(token: T): TokenAddressType[T] {
-    switch (token) {
-      case 'stETH':
-        return this.networkConfig.stEth as TokenAddressType[T];
-
-      case 'wstETH':
-        return this.networkConfig.wstEth as TokenAddressType[T];
-
-      case 'R':
-        return this.networkConfig.r as TokenAddressType[T];
-
-      default:
-        return null as TokenAddressType[T];
+  static getTokenAddress(token: Token): string {
+    try {
+      return this.networkConfig.tokenTickerToTokenConfigMap[token].address;
```

Comment:
You can use a `Record<Token, ...>` for `TokenTickerToTokenConfigMap` if you want to be sure that `TokenTickerToTokenConfigMap` will return non-`undefined` value. Also, it can be used as a type safe guard if we forget to add another token.